### ISSUE TITLE
Visibility: Add borders for table popovers; Buttons in dark mode blend with bg

### ISSUE THREAD

**Issue by kamilst96** [laravel-filament/filament#2818]
### Package

filament/filament

### Package Version

v2.13.5

### Laravel Version

v9.17

### Livewire Version

v2.10.5

### PHP Version

PHP 8.1.7

### Bug description

First, the filter popover blends into table header. Adding `border border-gray-300 dark:border-gray-700` and changing `mt-2` to `mt-4` in `filters/popover.blade.php` made it much more distinguishable.

![1](https://user-images.githubusercontent.com/37311362/174412626-22aafb56-b325-43a0-9b26-84dfef2e1dc2.png)

I also changed dark background to not blend with buttons `dark:bg-gray-800`.

![2](https://user-images.githubusercontent.com/37311362/174412628-5fab0cbb-b495-408b-bc7c-0db8a4faf8c3.png)

A second issue, with identical nature exists in tabs. I failed to hack it, because I didn't dig enough to find it yet, but the point is eliminating the buttons blending into background. It looks good on light, but not on dark. In this case, background could be darker, and inactive tabs brighter. The current feels like it's disabled.

![3](https://user-images.githubusercontent.com/37311362/174412785-8b7964bb-2dc1-438d-a435-1bd7dd596919.png)

Disabled DateTimePicker has focus outline.

![4](https://user-images.githubusercontent.com/37311362/174413444-8867e7d1-34db-4d3f-b1e2-9dfc5feaacf1.png)

Apologies for compound issue, but I wanted to avoid opening a few separate ones. I am not confident with theming, so I avoided opening PR. Can you intervene in PRs, when someone lacks skills to finish the job?

### Steps to reproduce

- Open table filter
- See DateTimePicker in View mode
- View tabs in dark mode

**Comment by kamilst96** [comment]
Okay, found how to do tabs. Here's my idea:
![5](https://user-images.githubusercontent.com/37311362/174414556-e0865f9b-8f74-4385-87ad-85dae8cf0103.png)

**Comment by danharrin** [comment]
Yeah! If you submit a PR, we can take over (as long as you are working on your own branch and not 2.x)
